src/main/java/com/google/api/codegen/discovery/transformer/php/PhpSampleMethodToViewTransformer.java

```diff
@@ -169,6 +169,8 @@ private SamplePageStreamingView createSamplePageStreamingView(
     builder.isResourceMap(fieldInfo.type().isMap());
     builder.pageVarName(
         symbolTable.getNewSymbol(namer.localVarName(Name.lowerCamel(fieldInfo.name()))));
+    builder.pageTokenName(methodInfo.requestPageTokenName());
```

Comment:
In PHP, the pageToken is assigned to an array by name, but the nextPageToken is accessed from an object by method ("getNextPageToken").